### DISCUSSION TITLE
API to create groups doesn't allow for multiple page rules

### DISCUSSION THREAD

**Discussion by CoffeeAddicts** [requarks/wiki#6437] [Error / Bug Report] [answered]
I'm having a pretty weird issue with the API. When adding a new group using the API the group will be added successfully however when I add multiple PageRules then more are added but the only with the characteristics of the first one. For Example:

```graph
mutation{
    groups{
      update(
        id:32
       name:"APITEST"
        redirectOnLogin:"Apps"
      	permissions:[
          "read:pages",
          "read:assets"]
        pageRules: [
          {
            id: "default"
            deny: false
            match:START
            roles: [
              "read:pages"
              "read:assets"
              
            ]
            path: "Apps/Test1"
            locales: []
          },
                
          {
            id: "default"
            deny: false
            match:START
            roles: [
              "read:assets"
            ]
            path: "Apps/Test2"
            locales: []
        }]
      )
			{
responseResult {
        succeeded
  	errorCode
        slug
        message
      }
      }
    }
  }
  ```

This should update a group and create two different page rules one with "Read:Assets" and "Read:Pages" in "Test1" and the other just "Read:Assests" in "Test2".

Upon using running this request. 2 page rules are added but they are just the first one duplicated.

![image](https://github.com/requarks/wiki/assets/104673207/2bd0cf46-c3af-4a1d-b1d3-b46ca086128f)

**Comment by NGPixel** [comment, marked as answer]
You're using the same ID twice...

**Reply by CoffeeAddicts** [reply]
I have no idea how I didn't notice that. Cheers.
